### PR TITLE
Update rpm spec

### PR DESCRIPTION
**Pull request by pkliczewski**
We need to make sure we declare dependencies as provided otherwise we fail to install on RHEL8.4.

Signed-off-by: Piotr Kliczewski <piotr.kliczewski@gmail.com>

### CODE REVIEW

File: k4e-agent.spec [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+Name:       k4e-agent
+Version:    1.0
+Release:    1%{?dist}
+Summary:    Agent application for the K4E (Kubernetes for Edge) solution
+
+Group:      K4E
+License:    GPL
+Source0:    https://github.com/jakub-dzon/%{name}-%{version}.tar.gz
+
+BuildRequires:  golang
```
Comment (pkliczewski) [inline k4e-agent.spec:12]: For RHEL we need few more dependencies:
* [gpgme-devel](http://mirror.centos.org/centos/8/PowerTools/x86_64/os/Packages/gpgme-devel-1.13.1-7.el8.x86_64.rpm)
*  [btrfs-progs-devel](https://mirror.rackspace.com/elrepo/testing/el8/x86_64/RPMS/btrfs-progs-devel-4.20.2-1.el8.x86_64.rpm)
* [btrfs-progs](https://mirror.rackspace.com/elrepo/testing/el8/x86_64/RPMS/btrfs-progs-4.20.2-1.el8.x86_64.rpm)
* [libbtrfs](https://mirror.rackspace.com/elrepo/testing/el8/x86_64/RPMS/libbtrfs-4.20.2-1.el8.x86_64.rpm)
* [libbtrfsutil](https://mirror.rackspace.com/elrepo/testing/el8/x86_64/RPMS/libbtrfsutil-4.20.2-1.el8.x86_64.rpm)
* [device-mapper-devel](http://mirror.centos.org/centos/8/PowerTools/x86_64/os/Packages/device-mapper-devel-1.02.175-5.el8.x86_64.rpm)
* [libassuan-devel](http://mirror.centos.org/centos/8/PowerTools/x86_64/os/Packages/libassuan-devel-2.5.1-3.el8.x86_64.rpm)

I haven't decide to update this yet

**Comment by pkliczewski** [comment]
Installation on rhel8.4 works but r4e iso creation fails :/

**Comment by pkliczewski** [comment]
`dnf clean all` did the trick. It was local cache issue :tada:

**Review by jakub-dzon — APPROVED** [review]

**Pull request merged**